use-timescale/services/service-management.md

```diff
@@ -91,6 +91,17 @@ digest algorithm 5) are cryptographic authentication mechanisms. $CONSOLE_LONG
 uses SCRAM by default. It is more secure and strongly recommended. The MD5
 option is provided for compatibility with older clients.
 
+## Change the $SERVICE_SHORT environment
+
+In $CLOUD_LONG, you can create $SERVICE_SHORTs for a development or production environment, and tag them accordingly: 
+
+- Development $SERVICE_SHORTs are suitable for less demanding tasks, such as ad hoc analytics, testing, or application building. Development $SERVICE_SHORTs have no delete protection. 
+- Production $SERVICE_SHORTs are suitable for mission-critical applications like client-facing dashboards or APIs. Production $SERVICE_SHORTs are protected from deletion. 
+
+$CLOUD_LONG applies upgrades differently to development and production $SERVICE_SHORTs. See [Maintenance and upgrades][maintenance-upgrades].
+
+To change the environment of an existing $SERVICE_SHORT, select it in the $CONSOLE_LONG, then click `Operations` > `Environment` and change the tag under `Change service environment`.
```

Comment:
To change the environment of an existing $SERVICE_SHORT:

1. In [$CONSOLE][services-portal], select the $SERVICE_SHORT to update
1. Click ...


\[services-portal]: https://console.cloud.timescale.com/dashboard/services